### ISSUE TITLE
failure to build

### ISSUE THREAD

**Issue by 0x210F** [haskell/haskell-platform#154]
While making an attempt to build a fresh checkout of the source tree (commit c0cdfe8a1728bf9eea5b7a2c44e7f7f0934091d9) on linux i received the following error:

Error when running Shake build system:
```
...
* build-all
* build-source
* build/product/haskell-platform-2014.2.0.0.tar.gz
* build/haskell-platform.cabal
More than one ghc version spec'd in release.
```

The following patch seems to have fixed the problem but I am not sure that this is the best fix:

```
diff --git a/hptool/src/Templates.hs b/hptool/src/Templates.hs
index 045a004..92e4ccc 100644
--- a/hptool/src/Templates.hs
+++ b/hptool/src/Templates.hs
@@ -74,7 +74,7 @@ expandRelease :: (Monad m) => Release -> MuContext m
 expandRelease rel = mkStrContext ex
   where
     ex "hpVersion" = MuVariable . showVersion . hpVersion . relVersion $ rel
-    ex "ghcVersion" = case pkgsThat [isGhc, not . isLib] of
+    ex "ghcVersion" = case pkgsThat [isGhc, not . isLib, not . isTool] of
         [] -> error "No ghc version spec'd in release."
         [ghcPkg] -> MuVariable . showVersion . pkgVersion $ ghcPkg
         _ -> error "More than one ghc version spec'd in release."

```

**Comment by rbtcollins** [comment]
FWIW this fixed master builds for me

**Comment by mzero** [comment]
yup - that was a bug!
Fixed in c3128dd922532328634d0d71ef5ef0e2c8f40282
